test/module/dy/fns/create.spec.js

```diff
@@ -214,4 +215,17 @@ describe('Testing create', {
     expect(error.message).to.equal('Validation failure on attribute(s): valid');
     expect(await getItemOrNull(key)).to.deep.equal(null);
   });
+
+  it('Testing create with timestamps enabled', async ({ capture }) => {
+    await generateTable({ timestamps: true });
+    const result = await model.create(item);
+    expect(result).to.deep.equal({ created: true, item });
+    const timestamps = await model.getItem(key, {
+      toReturn: ['created', 'modified']
+    });
+    expect(timestamps).to.deep.equal({
+      created: '2022-03-03T17:58:54.603Z',
+      modified: '2022-03-03T17:58:54.603Z'
+    });
+  });
```

Comment:
We have an issue. You cannot request `modified` or `created` on any function that internally uses `PUT`. This is because no attributes are returned from Dynamodb. we just return the attributes we provided. Since `modified` and `created` are managed by `dynamodb-toolbox` we don't have access to them.